standalone-metastore/pom.xml

```diff
@@ -89,7 +89,8 @@
     <log4j2.version>2.18.0</log4j2.version>
     <mockito-core.version>3.3.3</mockito-core.version>
     <orc.version>1.6.9</orc.version>
-    <protobuf.version>3.21.4</protobuf.version>
```

Comment:
After some checking, I think we should also update the protobuf in the root pom.xml

Reply:
@henrib looks like there is an unresolved comment. Has this been addressed?

Reply:
Updated root pom.xml in new commit